src/main/java/org/jetbrains/bsp/bazel/server/BazelBspServer.java

```diff
@@ -610,14 +608,9 @@ private String getSourcesRoot(String uri) {
     for (Map.Entry<String, String> diagnostics : diagnosticsProtosLocations.entrySet()) {
```

Comment:
it can be streamified

Reply:
Right, but it's _BazelBspServer_, there's much more to be done there later.

Reply:
an not in this PR

Reply:
Yeah, that's what I meant